jenkins/pull_request_in_docker32.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
This file is mostly the same as pull_request_in_docker.sh except for the actual test list. I think we should use the same file and use an ENV variable to control the test set.